examples/nginx-grafana-metabase/README.md

```diff
@@ -0,0 +1,9 @@
+# Visualizing Nginx Access Logs using Grafana and Metabase
+In my last [blog post](https://www.timeplus.com/post/analyzing-nginx-access-logs), I showed how Timeplus Proton can be used to analyze web traffic in production using just the log files generated by an Nginx web server. The article explored real-time analysis and historical analysis but the analysis wasn't visualized. 
```

Comment:
Hi Saïd, thank you for writing this article! Here are some minor wording edits: 

... web traffic in production, using just the log files... 

- [ ] Add a comma after "in production"

Reply:
Done.